SLHCUpgradeSimulations/Configuration/python/phase2TkFlat.py

```diff
@@ -27,6 +31,129 @@ def customise(process):
 
     return process
 
+
+def customise_SimTkOnly(process):
```

Comment:
Technically easy (although I don't understand what exactly `process.simulationTkOnly_step.remove(process.g4SimHits.MuonSD)` etc tries to achieve, AFAICT it is a no-op).

With eras, the same effect as `del process.g4SimHits.MuonSD` can be achieved with
```
# in SimG4Core/Application/python/g4SimHits_cfi.py
eras.<sub-era name>.toModify(
    MuonSD = None,
    CaloSD = None,
    CaloResponse = None,
    ... # etc
)
```

The `customise_DigiTkOnly` function needs a bit more, but everything should be doable with `toModify()`. Hmm, some of it is a copy-paste from `customise_Digi`, and there turning `customise_DigiTkOnly` to eras would be easier after(/at the same time as) `customise_Digi` is turned to eras.

Then the question is what `<sub-era name>` should be. I guess `Configuration/StandardSequences/python/Eras.py` would need something along (feel free to invent better names)
```
self.phase2_tracker_only = cms.Modifier()
...
self.Phase2LReco_trackerOnly = cms.ModifierChain(self.Phase2LReco, self.phase2_tracker_only)
# assuming Phase2GReco with tracker-only is not needed ATM
```
Writing this down actually makes me feel a bit dirty ;), but I let others decide if this is an acceptable solution.